src/app/App.tsx

```diff
@@ -6,24 +6,35 @@ import {ThemeProvider} from "@material-ui/core/styles";
 import {CssBaseline} from "@material-ui/core";
 import {theme} from "./theme/Theme"
 import {GlobalStyleProvider} from "./theme/GlobalStyleProvider";
+import Grid from '@material-ui/core/Grid';
 
 export const App = () => {
     document.title = "Stat Check"
     return (
         <ThemeProvider theme={theme}>
             <GlobalStyleProvider>
                 <CssBaseline/>
-                <Router>
-                    <Header/>
-                    <Switch>
-                        <Route path="/champion">
-                            <ChampionPage/>
-                        </Route>
-                        <Route path="/">
-                        </Route>
-                    </Switch>
-                </Router>
+                <Header/>
+                <Grid container spacing={3} justify='center'>
+                    <Grid item md={4}>
+                        <Routes/>
+                    </Grid>
+                </Grid>
             </GlobalStyleProvider>
         </ThemeProvider>
     );
 }
+
+const Routes = () => {
```

Comment:
you can move Routes in a specific package but as you wish. It's not a bad place anyway.

Reply:
I personnaly don't mind having them here, as for now there isn't a lot of them. But the change is possible indeed